src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -28,7 +57,14 @@ internal class SentryXcodeProject : IDisposable
         private readonly INativeMain _nativeMain;
         private readonly INativeOptions _nativeOptions;
 
-        public SentryXcodeProject(string projectRoot) : this(projectRoot, new NativeMain(), new NativeOptions())
+        static SentryXcodeProject()
+        {
+            var xcodeAssembly = Assembly.Load("UnityEditor.iOS.Extensions.Xcode");
```

Comment:
I believe this will throw at first access to this type (`SentryXcodeProject`).
In which case we need to handle error here and make sure `Type` isn't null in the instance constructor.

Do we have a test in CI that validates this with and without the iOS build tools? I'd expect a broken build

Reply:
We run the smoke test on Windows without the iOS module but no one is accessing the type. I've created a follow up issue so we won't hit that issue again: https://github.com/getsentry/sentry-unity/issues/610